easy_phi/app.py

```diff
@@ -208,9 +208,12 @@ class SelectModuleHandler(ModuleHandler):
     def post(self):
         """ Set user lock on module to indicate it is used by someone """
         used_by = getattr(self.module, 'used_by', None)
+        if used_by == auth.user_by_token(self.api_token):
+            self.finish("OK")
+            return
         if used_by is not None:
             self.set_status(400)
-            self.write({'error': "Module is used by {0}. If you need this "
+            self.finish({'error': "Module is used by {0}. If you need this "
```

Comment:
this is not functional change, just a defensive practice. If return is missed, next .write() will raise an Exception